components/Post/Post.jsx

```diff
@@ -148,6 +148,7 @@ export default class Post extends React.Component {
       data.likes--;
       data.usersLiked = _.filter(data.usersLiked, user => user._id !== loggedInUser._id);
     } else {
+      data.isLiked = true;
```

Comment:
Should probably also add `data.isLiked = false;` under `if (data.likes < 0)` below (that should probably be a <= too)